examples/reservation-manager/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+#![no_std]
+
+use gstd::{msg, prelude::*, ReservationManager};
+
+static mut RESERVATION_MANAGER: Option<ReservationManager> = None;
+
+#[no_mangle]
+extern "C" fn handle() {
+    let new_msg = String::from_utf8(msg::load_bytes().expect("Failed to load payload bytes"))
```

Comment:
We recently found encoding issue. Please replace this code with the following: https://github.com/gear-dapps/ping/commit/4001429180e0aabe2888ecdd22f8d82a564f0edc
See https://github.com/gear-tech/gear-js/issues/1280